sql/core/src/test/scala/org/apache/spark/sql/DataFrameAggregateSuite.scala

```diff
@@ -612,7 +615,9 @@ class DataFrameAggregateSuite extends QueryTest with SharedSparkSession {
          Seq((true, true), (true, false), (false, true), (false, false))) {
       withSQLConf(
         (SQLConf.WHOLESTAGE_CODEGEN_ENABLED.key, wholeStage.toString),
-        (SQLConf.USE_OBJECT_HASH_AGG.key, useObjectHashAgg.toString)) {
+        (SQLConf.USE_OBJECT_HASH_AGG.key, useObjectHashAgg.toString),
+        (SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "false")) {
+        // When enable AQE, the WholeStageCodegenExec is added during QueryStageExec.
```

Comment:
Isn't this a bug? Maybe the `conf` passed in `AdaptiveSparkPlanExec` is not right?